web/src/compose_state.ts

```diff
@@ -92,6 +101,12 @@ export function composing(): boolean {
     return Boolean(message_type);
 }
 
+export let is_item_clicked = (): boolean => compose_select_recipient_dropdown_widget.item_clicked;
```

Comment:
That's not a good function name. It doesn't describe what was clicked. Looks like we are using this `.item_clicked` in other places too. Can you refactor those other places to use this function too and add a prep commit for it.

Reply:
`is_recipient_dropdown_item_clicked` should be clear enough.